releasenotes/notes/primitives-v2-df871c0c6ac0b94a.yaml

```diff
@@ -0,0 +1,157 @@
+---
+features:
+  - |
+    Version 2 of the primitives was introduced via a new base class for both the sampler and the
+    estimator, along with new types for their inputs and outputs. The emphasis of this new
+    version is on performing vectorized calls to the primitive ``run()`` methods, so that sweeps
+    over parameter value sets and observables can be efficiently specified. See
+    :class:`~.StatevectorSampler` and :class:`~.StatevectorEstimator` for reference implementations
+    of the V2 primitives.
+
+    Moreover, the estimator has gained a ``precision`` argument in the :meth:`~.BaseEstimatorV2.run`
+    method that specifies the targeted precision of the expectation value estimates. Analagously,
+    the sampler has moved ``shots`` out of the options and into the arguments of the
+    :meth:`~.BaseSamplerV2.run` method. The sampler has also been changed to return the outputs
+    (e.g. bitstrings) from every shot, rather than providing a :class:`~.Counts`\-like return, and
+    also to store data from separate :class:`~.ClassicalRegister`\s . This enables derived classes
+    to implement sampler support for circuits with classical control flow.
+    
+    The primitive V2 base classes are:
+
+    * :class:`.BaseSamplerV2`
+    * :class:`.BaseEstimatorV2`
+
+    The new types which are used for inputs and outputs are:
+
+    * :class:`.SamplerPubLike`\: primitive unified bloc (PUB) of sampler inputs; a union type
```

Comment:
I think these type ones are `const` not `class`, and im not sure they show up in the docs atm